### DISCUSSION TITLE
Unhandled exception

### DISCUSSION THREAD

**Discussion by askoa** [onekey-sec/unblob#915] [Q&A] [answered]
I ran unblob using docker container (I use podman but that should not have any effect) and get the below error. I added `-vvv` option to increase the verbosity but did not see any change in the output.

command:
```
podman run --rm --pull always -v /path/to/file:/data/input:z -v /path/to/output:/data/output:z   ghcr.io/onekey-sec/unblob:latest -vvv /data/input/flash.bin
```
error:
```
{'exc_info': '0x1', 'event': 'Unhandled exception during unblob', 'level': 'error', 'timestamp': '2024-08-01 11:09.23', 'pid': 1}
```

**Comment by qkaiser** [comment, marked as answer]
Dupe of https://github.com/onekey-sec/unblob/issues/959 which is related to bind mount directories and write permissions.
